aztk/client.py

```diff
@@ -225,6 +225,32 @@ def __delete_user_on_pool(self, username, pool_id, nodes):
             futures = [exector.submit(self.__delete_user, pool_id, node.id, username) for node in nodes]
             concurrent.futures.wait(futures)
 
+    def __node_run(self, cluster_id, node_id, command, internal, container_name=None, timeout=None):
+        pool, nodes = self.__get_pool_details(cluster_id)
+        node = [node for node in nodes if node.id == node_id]
+        if node:
+            node = node[0]
+        if internal:
+            node_rls = models.RemoteLogin(ip_address=node.ip_address, port="22")
+        else:
+            node_rls = self.__get_remote_login_settings(pool.id, node.id)
+        ssh_key = RSA.generate(2048)
+        ssh_pub_key = ssh_key.publickey().exportKey('OpenSSH').decode('utf-8')
+        try:
+            self.__create_user_on_node('aztk', pool.id, node.id, ssh_pub_key)
```

Comment:
shouldn't this user have some kind of random id so it doesn't conflict

Reply:
maybe, but if the user already exists, then it is just deleted and recreated. So the only potential issue is if someone adds `aztk` as a user, and then runs a command. In that case, their created user would be deleted, so adding a randomly generated user id would probably be best here.

Reply:
or running node-run twice at the same time. Not sure what deleting the user while being login does

Reply:
I believe nothing would happen while the session is open. However, still, making the user id generated would solve that.